justfile

```diff
@@ -348,10 +348,11 @@ wipe-prometheus:
     rm -rf data
 
 
+alias e2e := tests-e2e
 # end-to-end tests
-e2e: services
+tests-e2e: services
     #!/usr/bin/env bash
     set -euxo pipefail
-    cargo test -- --test-threads=1
+    cargo test -p tests-e2e -- --ignored --test-threads=1 --nocapture
```

Comment:
Not sure if we want `--nocapture` by default. Can we pass it in as an argument to the recipe instead?

Reply:
added the argument (also aligned ln-dlc-node-tests)